finagle-core/src/main/scala/com/twitter/finagle/ssl/SslConfigurations.scala

```diff
@@ -28,6 +31,13 @@ private[ssl] object SslConfigurations {
           case Return(kms) => Some(kms)
           case Throw(ex) => throw SslConfigurationException(ex.getMessage, ex)
         }
+      case KeyCredentials.KeyAndCertChain(key, certChain) =>
```

Comment:
Undo this change please. If you would like to add this outside the context of this pull request, we can discuss it in a separate PR.